src/main/scala/org/broadinstitute/clio/dataaccess/HttpElasticsearchDAO.scala

```diff
@@ -130,12 +131,17 @@ object HttpElasticsearchDAO extends LazyLogging {
   private def fieldDefinition(elasticsearchField: ElasticsearchField): FieldDefinition = {
```

Comment:
Exactly! The `ElasticsearchField` was at one point an elastic4s `FieldDefinition`, but Phil pointed out that the Elasticsearch specific code should be hidden away. Now, `ElasticsearchField` might as well be called `TableColumn`-- there's nothing particularly ES specific about the data stored in each instance.